src/index.test.js

```diff
@@ -95,6 +95,28 @@ describe('LazyRender', () => {
       expect(wrapper.prop('className')).toEqual('my-cool-class');
     });
 
+    [
+      {
+        appearance: 'inline',
+        expectedNode: 'span'
+      },
+      {
+        appearance: 'block',
+        expectedNode: 'div'
+      },
+    ].forEach(item => {
```

Comment:
/nit I prefer being explicit with test and duplicate code instead of using a forEach